capa/main.py

```diff
@@ -599,6 +565,9 @@ def handle_common_args(args):
 
 
 def main(argv=None):
+    if sys.version_info < (3, 6):
+        raise UnsupportedRuntimeError("capa should only be used with Python 3.6+")
```

Comment:
```suggestion
        raise UnsupportedRuntimeError("This version of capa can only be used with Python 3.6+")
```

Reply:
Thoughts on clarifying this as suggested?

Reply:
bump

Reply:
oh! I forgot this. I don't have a strong opinion about this, so I was waiting in case someone else had something to say. I can change it.